packages/ui/src/table/TableHeaderCell.tsx

```diff
@@ -51,6 +51,7 @@ export const TableHeaderCell = ({
         {Boolean(sort) && (
           <Box paddingX={2}>
             <IconButton
+              accessibilityHint=""
```

Comment:
I'm guessing this is updated in the larger table PR?

Reply:
Correct!